docs/hpc/03_storage/03_data_transfers.md

```diff
@@ -1 +1,75 @@
+-   From the Greene cluster (e.g., the login nodes): the hostname can be shortened to gdtn
+-   For example, to log in to a DTN from the Greene cluster, to carry out some copy operation, and to log back out, you can use a command sequence like:
+```sh
+ssh gdtn
+rsync ...
+logout
+```
+-   Via specific tools like Globus (see below)
+
+## Linux & Mac Tools
+### scp and rsync
+Please transfer data using Data-Transfer nodes
+
+Sometimes these two tools are convenient for transferring small files. Using the DTNs does not require to set up an SSH tunnel; use the hostname dtn.hpc.nyu.edu for one-step copying. See below for examples of commands invoked on the command line on a laptop running a Unix-like operating system:
+```sh
+scp HMLHWBGX7_n01_HK16.fastq.gz jdoe55@dtn.hpc.nyu.edu:/scratch/jdoe55/
+rsync -av HMLHWBGX7_n01_HK16.fastq.gz jdoe55@dtn.hpc.nyu.edu:/scratch/jdoe55/ 
+```
+In particular, rsync can also be used on the DTNs to copy directories recursively between filesystems, e.g. (assuming that you are logged in to a DTN),
+```sh
+rsync -av /scratch/username/project1 /rw/sharename/
+```
+where username would be your user name, project1 a directory to be copied to the Research Workspace, and sharename the name of a share on the Research Workspace (either your NetID or the name of a project you're a member of).
+
+## Windows Tools
+### File Transfer Clients
+Windows 10 machines may have the Linux Subsystem installed, which will allow for the use of Linux tools, as listed above, but generally it is recommended to use a client such as [WinSCP](https://winscp.net/eng/docs/tunneling) or [FileZilla](https://filezilla-project.org/) to transfer data. Additionally, Windows users may also take advantage of [Globus](./04_globus.md) to transfer files.
+
+### Tunneling
+[Read the detailed instructions for setting up tunnels.](../02_connecting_to_hpc/02_ssh_tunneling_and_x11_forwarding.md)
+
+## Globus
+Globus is the recommended tool to use for large-volume data transfers. It features automatic performance tuning and automatic retries in cases of file-transfer failures. Data-transfer tasks can be submitted via a web portal. The Globus service will take care of the rest, to make sure files are copied efficiently, reliably, and securely. Globus is also a tool for you to share data with collaborators, for whom you only need to provide the email addresses.
+
+The Globus endpoint for Greene is available at `nyu#greene`. The endpoint `nyu#prince` has been retired.
+
+[Please see detailed instructions](./04_globus.md)
+
+## rclone
+rclone - rsync for cloud storage, is a command line program to sync files and directories to and from cloud storage systems such as Google Drive, Amazon Drive, S3, B2 etc. rclone is available on DTNs.
+
+[Please see the documentation for how to use it.](https://rclone.org/)
+
+## Open OnDemand
+One can use Open OnDemand interface to upload data.
+However, please use it only for small data!
+
+## Other tools
```

Comment:
Convert this to a tip instead of a new Other tools section?

Reply:
Hmm, as I look at this again it seems that the introduction is just another one of those list summaries at the tops of the pages we've been removing.  How about I remove the entire Introduction section and move the warning to the end to the Data-Transfer nodes section?
Or maybe at the start of the Data-Transfer nodes section?
What do you think @s-sajid-ali ?

Reply:
And I'll make the Other tools section a tip as you suggested.

Reply:
>Hmm, as I look at this again it seems that the introduction is just another one of those list summaries at the tops of the pages we've been removing. How about I remove the entire Introduction section and move the warning to the end to the Data-Transfer nodes section?

Yeah that sounds good to me. I'd also suggested that we make the data management page the landing page for the section. Maybe we can add a short overview to that page if we make it the landing page?

Reply:
Great idea.  I just added it after the current into material with a little modification.